pylint/reporters/reports_handler_mix_in.py

```diff
@@ -12,7 +12,7 @@
 if TYPE_CHECKING:
     from pylint.lint.pylinter import PyLinter
 
-ReportsDict = DefaultDict[IChecker, List[Tuple[str, str, Callable]]]
+REPORTS_DICT = DefaultDict[IChecker, List[Tuple[str, str, Callable]]]
```

Comment:
Type aliases are usually PascalCase. So the existing ones are correct.

Reply:
My bad, I thought this was only for `TypedDict` and `NamedTuple` as they can use `class` to instantiate. Going to close!